### DISCUSSION TITLE
Checkerboard texture generator

### DISCUSSION THREAD

**Discussion by seacat17** [godotengine/godot-proposals#9411] [3D]
It is as simple as it sounds. I need a checkerboard pattern on the floor, but to make it I need to hop into GIMP and make it myself. Why can't Godot generate it for me like it generates noise or gradients?

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in:
* https://github.com/godotengine/godot-proposals/discussions/5294
